docs/en/loading/loading_introduction/Loading_intro.md

```diff
@@ -0,0 +1,199 @@
+---
+displayed_sidebar: "English"
+toc_max_heading_level: 5
+---
+
+# Loading options
+
+Data loading is the process of cleansing and transforming raw data from various data sources based on your business requirements and loading the resulting data into StarRocks to facilitate analysis.
+
+StarRocks provides a variety of options for data loading:
+
+- Loading methods: Insert, Stream Load, Broker Load, Pipe, Routine Load, and Spark Load
+- Ecosystem tools: StarRocks Connector for Apache Kafka® (Kafka connector for short), StarRocks Connector for Apache Spark™ (Spark connector for short), StarRocks Connector for Apache Flink® (Flink connector for short), and other tools such as SMT, DataX, CloudCanal, and Kettle Connector
+- API: Stream Load transaction interface
+
+These options each have its own advantages and support its own set of data source systems to pull from.
+
+This topic provides an overview of these options, along with comparisons between them to help you determine the loading option of your choice based on your data source, business scenario, data volume, data file format, and loading frequency.
+
+## Introduction to loading options
+
+This section mainly provides the characteristics and business scenarios of the loading options available in StarRocks.
+
+![Loading options overview](../../assets/loading_intro_overview.png)
+
+:::note
+
+In the following sections, "batch" or "batch loading" refers the loading of a large amount of data from a specified source all at a time into StarRocks, whereas "stream" or "streaming" refers to the continuous loading of data in real time.
+
+:::
+
+### Loading methods
```

Comment:
```suggestion
## Loading methods
```

Reply:
No. Actually "loading options" consists of three parts, which are "loading methods", "ecosystem tools", and "API".

Reply:
No, loading options is four parts: intro, methods, tools, and API. Why would you consider methods to be a component of intro? Same question for tools and API?